src/main/java/org/wso2/carbon/connector/azure/storage/BlobUploader.java

```diff
@@ -55,8 +55,14 @@ public void connect(MessageContext messageContext) {
         String fileName = messageContext.getProperty(AzureConstants.FILE_NAME).toString();
         String filePath = messageContext.getProperty(AzureConstants.FILE_PATH).toString();
 
+        String protocol = "http";
+        if (messageContext.getProperty(AzureConstants.PROTOCOL) != null) {
```

Comment:
same here, and shall we have "http" in a cosntant.

Reply:
I guess, we can write a small util to get protocol, passing msgCtx and re use, since this is used in many places.

Reply:
seems we can get storageConnectionString itself from util ?